url_screenshot/icon_url_screenshot/actions/screenshot/action.py

```diff
@@ -0,0 +1,59 @@
+import komand
+from .schema import ScreenshotInput, ScreenshotOutput, Input, Output, Component
+# Custom imports below
+from komand.exceptions import PluginException
+import os
+import uuid
+import base64
+import time
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+
+class Screenshot(komand.Action):
+    CHROME_PATH = '/usr/bin/chromium'
+    CHROMEDRIVER_PATH = '/usr/bin/chromedriver'
+    WINDOW_SIZE = "1920,1080"
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='screenshot',
+                description=Component.DESCRIPTION,
+                input=ScreenshotInput(),
+                output=ScreenshotOutput())
+
+    def run(self, params={}):
+
+        name = f"/tmp/{str(uuid.uuid1())}.png"
```

Comment:
A cleaner way to work with the file writing/reading in this function would be to utilize the tempfile library bundled with Python. Example:
```
with tempfile.NamedTemporaryFile(suffix=".png") as t:
    # Print the path to the file, for example
       print(t.name)

>>>> /var/folders/7g/64z6m5x92n12p21mspg9d384xwrlwt/T/tmp0bkf6n14
```

No need to write to the filesystem and the file will automatically be removed from memory after the context closes

Reply:
Nice!  I'll implement